setup.py

```diff
@@ -66,6 +66,7 @@
         "matplotlib",
         "numkit",  # numerical helpers
         "numpy>=1.0",
+        "setuptools",  # contains 'pkg_resources' for versioneer
```

Comment:
Only require setuptools for Python>=3.12:
```suggestion
        'setuptools; python_version >= "3.12"',  # contains 'pkg_resources' for versioneer
```

Reply:
Thanks @njzjz , I added you to AUTHORS.